addons/website_project_issue/controllers/main.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import werkzeug
+
+from openerp import http
+from openerp.http import request
+
+from openerp.addons.website_portal.controllers.main import website_account
+
+class WebsiteAccount(website_account):
+    @http.route(['/my', '/my/home'], type='http', auth="user", website=True)
+    def account(self):
+        response = super(WebsiteAccount, self).account()
+        project_issues = request.env['project.issue'].search([])
```

Comment:
Obviously, this is not correct. A search without domain and without limit looks terrible at first glance!
Why not searching on issues where the user is a contact person? We do not want to display order issues. (Check also with public project, I'm not sure the record rules will give you what is expected in this task)